Config/Jabo.ini

```diff
@@ -4731,22 +4731,25 @@ Internal Name=Neon64 by HCS
 Counter Factor=Default
 Good Name=Nintendo On My Mind Demo by Kid Stardust (PD)
 Internal Name=Nintendo On My Mind
-
+
 [D1C6C55D-F010EF52-C:00]
-�[4E01B4A6-C884D085-C:00]

```

Comment:
This stray leading non-ASCII character was causing a trap when running in the VS debugger. LOL